spring-remoting/remoting-http/client/src/main/java/com/baeldug/client/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldug.client;
+
+import com.baeldung.api.CabBookingService;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.remoting.httpinvoker.HttpInvokerProxyFactoryBean;
+
+@Configuration
+public class Main {
+
+    @Bean
+    public HttpInvokerProxyFactoryBean invoker() {
+        HttpInvokerProxyFactoryBean invoker = new HttpInvokerProxyFactoryBean();
+        invoker.setServiceUrl("http://localhost:9090/spring-remoting-http-server/account");
```

Comment:
When I first ran it locally, I got a 404 error; it seems that the URL (as specified in the service bean) should be "/booking" instead of "/account". When I changed this locally, it worked. Note: the code snippet in the article seems to be correct, but please double-check it after you make this correction.